lib/sort_by_length.rb

```diff
@@ -1,7 +1,22 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2)?
+# Space complexity: 0(1)?
 def sort_by_length(my_sentence)
```

Comment:
Well done bubblesort, but because you're doing a `.split`, you have O(n) space complexity.